utils/db/handle.js

```diff
@@ -0,0 +1,27 @@
+import firebase from "../firebase/firebase";
+const _ = require('lodash');
+
+export async function getTeamsForUserId(userId) {
+  let teams = []
+
+  const teamIds = await
+    firebase
+      .database()
+      .ref("/users/" + userId)
+      .once("value")
+      .then((returned) => {
+        return returned.val().teams
+      });
+  
+  teams = await Promise.all(_.map(teamIds, teamId => { //Using Promise.all avoids making the _.map an async function
```

Comment:
Can replace with `teamIds.map(teamId => { ... })` See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
No need for lodash in this case